GitUI/UserControls/BlameControl.cs

```diff
@@ -427,6 +440,38 @@ private void contextMenu_Opened(object sender, EventArgs e)
             blamePreviousRevisionToolStripMenuItem.Enabled = selectedRevision.HasParent;
         }
 
+        private void ConfigureViewInGitHubMenu()
```

Comment:
This could be extended to other sites, suggest changing GitHub to Website

Reply:
done.